packages/holocron/docs/api/README.md

```diff
@@ -613,6 +615,52 @@ const getModulesToUpdate = (
 
 <!--ONE-DOCS-ID end-->
 
+### Advanced Functions
+
+These functions are exposed to allow more advanced uses of Holocron.
+
+<!--ONE-DOCS-ID id="forceLoadModule" start-->
+
+#### `forceLoadModule`
+
+Re-loads a modules source code, re-injects it into the page, and return its top level export.
+
+This function can allow a browser to re-fetch a module directly, without requiring a page load.
+
+Note: you should not need to call this function yourself to load modules into the browser under most use cases. Instead, use the duck `loadModule`.
+
+##### Arguments
+
+| name | type | required | value |
+|---|---|---|---|
+| `moduleName` | `string` | `true` | The name of the module to try to reload |
+| `moduleData` | `object` | `true` | The moduleData object that stores the url, and integrity for all modules |
+
+
+##### Usage
+
+Directly load a fresh instance of a module. Note: Under normal use cases, use RenderModule to render a module in JSX.
+
+```js
+import { forceLoadModule, getModuleMap } from 'holocron';
+
+export const MyComponent = (props) => {
+  const [LoadedModule, setLoadedModule] = React.useState(null);
```

Comment:
Doesn't `LoadedModule` need to be initialized as a function?

Reply:
Becasue we are rendering as 
```jsx
return LoadedModule || (
    <LoadedModule {...props} />
  );
```

null is fine. I was tempted to put a loading state in, but that goes beyond the simple example.